packages/protocolbeat/src/colors.json

```diff
@@ -23,6 +23,7 @@
     "yellow": "#FABD30",
     "cyan": "#1c92a8",
     "pink": "#e27991",
-    "purple": "#a73db5"
+    "purple": "#a73db5",
+    "grey": "#928474"
```

Comment:
Can't we use the same color we use for comments in the editor?

Reply:
Good point, fixed in the last commit